src/rightPanel/RightPanel.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import KeyboardTabIcon from "@mui/icons-material/KeyboardTab";
+import { IconButton } from "@mui/material";
+import ValueSetsSection from "./valueSetsTab/ValueSetsSection";
+import RightPanelNavTabs from "./RightPanelNavTabs";
+import DefinitionsSection from "./definitionsTab/DefinitionsSection";
+import CodesSection from "./codesTab/CodesSection";
```

Comment:
in other projects I think we followed convention- same folder name as conponent. e.g.`"./codesSection/CodesSection";`

Reply:
thank you, i have updated it